run-tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
```

Comment:
I also often add `-u` (`set -eu`) to ensure all variables are defined

Reply:
I meant to have `set -e` in one place, and `set -u` in another, to allow `$1` to be empty. Thanks for catching this

Reply:
A trick is to use `${1-}` to bypass this.

Reply:
Cool